rewards/utils/rewards_utils.py

```diff
@@ -78,6 +79,32 @@ def distribute_rewards_to_snapshot(
     return combine_rewards([rewards] + custom_rewards_list, 0)
 
 
+def distribute_rewards_from_total_snapshot(
+        amount: int, snapshot: Snapshot, token: str,
+        custom_rewards: Optional[Dict[str, Callable]] = None,
+):
+    if not custom_rewards:
+        custom_rewards = {}
```

Comment:
Why not define like this?
```
def distribute_rewards_from_total_snapshot(
        amount: int, snapshot: Snapshot, token: str,
        custom_rewards: Optional[Dict[str, Callable]] = {},
):
```

Reply:
Please, don't pass default mutable arguments to functions. This will lead to super tricky bugs. 

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Gotcha, I don't think the example is exactly applicable here but good to know the pattern moving forward.